walletjs-cli/index.js

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env node
+
+const os = require("os");
+const fs = require("fs");
+const { Command } = require("commander");
+const core = require("@elrondnetwork/elrond-core-js");
+
+main();
+
+function main() {
+    const program = new Command();
+    program.version("1.0.0");
+    setupCli(program);
+
+    try {
+        program.parse(process.argv)
+    } catch (error) {
+        console.error(error.toString());
+        throw error;
+    }
+}
+
+function setupCli(program) {
+    program
+        .command("generate-mnemonic")
+        .description("Generate a new mnemonic phrase (24 words) and write it to a file")
+        .requiredOption("-o, --out-file <outFile>", "where to write the mnemonic")
+        .option("-d, --display", "also display mnemonic in stdout")
+        .action(generateMnemonic);
+
+    program
+        .command("generate-key-file")
```

Comment:
In order not to confuse people I would make only one command, generate-account, that will generate mnemonic then key-file given that mnemonic. The mnemonic would be printed in the console with a message to save it somewhere safe